app/players/elves/druid.py

```diff
@@ -0,0 +1,13 @@
+from .elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, nickname: str, musical_instrument: str, favourite_spell: str):
+        super().__init__(nickname, musical_instrument)
+        self.favourite_spell = favourite_spell

```

Comment:
Make sure that you store arguments in the protected attributes as mentioned in the checklist